src/ScintillaWnd.cpp

```diff
@@ -2410,20 +2379,6 @@ void CScintillaWnd::ReflectEvents(SCNotification* pScn)
     switch (pScn->nmhdr.code)
     {
         case SCN_PAINTED:
-            if (m_LineToScrollToAfterPaint != -1)
-            {
-                auto visLineToScrollTo = Call(SCI_VISIBLEFROMDOCLINE, m_LineToScrollToAfterPaint);
-                visLineToScrollTo += m_WrapOffsetToScrollToAfterPaint;
-                Call(SCI_SETFIRSTVISIBLELINE, visLineToScrollTo);
-                --m_LineToScrollToAfterPaintCounter;
-                if ((m_LineToScrollToAfterPaintCounter <= 0) || ((Call(SCI_GETFIRSTVISIBLELINE) + 2) > Call(SCI_VISIBLEFROMDOCLINE, m_LineToScrollToAfterPaint)))
-                {
-                    m_LineToScrollToAfterPaint        = -1;
-                    m_WrapOffsetToScrollToAfterPaint  = 0;
-                    m_LineToScrollToAfterPaintCounter = 0;
-                }
-                UpdateLineNumberWidth();
-            }
```

Comment:
When wrapping is enabled, all the "go to line" commands in Scintilla won't work properly because the wrapping info is only calculated when the scintilla view is painted. That's why this code is here.
Why did you remove this?

Reply:
I removed the code because I am *hoping* the code I removed is working around a problem the old code created and which I am *hoping* the new code fixes. I'm not seeing any wrapping problems yet, not that It would surprise me as I don't claim to be expert here as my comment says. What new problem do you see here related to wrapping?